### PR TITLE
ENH: update logic to handle deprecated error logs, log deletion boolean option [GEAR-348]

### PR DESCRIPTION
**Pull request by kalebdfischer**

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -355,6 +378,7 @@ def get_errors(error_containers, client):
         if error_log_filenames:
             for error_log_filename in error_log_filenames:
                 origin_file_dict = get_error_origin_file_dict(container.to_dict(), error_log_filename)
+                log.critical('Reading file %s on %s %s', error_log_filename, container.container_type, container.id)
```
Comment (NPann) [inline run.py:381]: Why the critical level here?

File: run.py [hunk 2]
```diff
@@ -336,7 +359,7 @@ def get_error_origin_file_dict(container_dict, error_log_name):
     return file_dict
 
 
-def get_errors(error_containers, client):
+def get_errors(error_containers, client, delete_errors=False):
```
Comment (NPann) [inline run.py:362]: Nice addition :) 
Nit: add delete_errors to docstring?

File: run.py (hunk 1 above)
Comment (kalebdfischer) [reply run.py]: [4c1b77e](https://github.com/flywheel-apps/GRP-2/pull/16/commits/4c1b77e9657ea01af0bef9eaba986f75e7458a3f)

File: run.py (hunk 2 above)
Comment (kalebdfischer) [reply run.py:362]: [9a5b387](https://github.com/flywheel-apps/GRP-2/pull/16/commits/9a5b3871d1f27336cd738b7305dd34450451ae7d)

**Pull request merged**